static/src/javascripts/.eslintrc.js

```diff
@@ -1,84 +1,5 @@
 module.exports = {
-    env: {
-        browser: true,
-        jest: true,
-    },
-    parser: 'babel-eslint',
-    parserOptions: {
-        sourceType: 'module',
-        ecmaVersion: 6,
-    },
     settings: {
         'import/resolver': 'webpack',
     },
-    extends: [
-        'prettier/react',
-    ],
-    rules: {
-        // require-specific overrides
-        'import/extensions': 'off',
-        'import/no-webpack-loader-syntax': 'off', // used for require plugins still
-        'import/no-namespace': 2,
-        'import/prefer-default-export': 'off', // we explicitly warn against default export below
-
-        // these are bad habits in react that we're already abusing.
-        // if we go more [p]react we should look at them.
-        // not saying it's ok, but we don't reuse modules or
-        // develop this stuff much. disabling for now.
-        'react/no-multi-comp': 'off',
```

Comment:
We lint React but the guardian configs eslint doesn't - do we want to keep this here or should there be a react eslint config at the guardian level?

Reply:
I reckon we should do a `@guardian/eslint-react`?